bindata/oauth-openshift/deployment.yaml

```diff
@@ -6,6 +6,12 @@ metadata:
   labels:
     app: oauth-openshift
 spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      # make sure at least one pod is always active in HA clusters
+      maxUnavailable: 2
```

Comment:
this is what we have for `oauth-apiserver`

```
spec:
  strategy:
    type: RollingUpdate
    rollingUpdate:
      # To ensure that only one pod at a time writes to the node's
      # audit log, require the update strategy to proceed a node at a
      # time. Only when a master node has its existing
      # oauth-apiserver pod stopped will a new one be allowed to
      # start.
      maxUnavailable: 1
      maxSurge: 0
```

Reply:
the oauth-server does not write to any node-owned file, although - do you propose only one at a time be unavailable? I am not sure what the more reasonable choice would be, I suppose forcing at most 1 pod unavailable might be better?

Reply:
since the number of masters can be dynamic and not always == `3` I think setting it to `1` is more preferable.